bloom/generators/debian/generator.py

```diff
@@ -34,6 +34,7 @@
 
 import collections
 import datetime
+import io
```

Comment:
I had initially imported `io.open` directly into the local module but that caused every other instance of `open` to want `unicode`s instead of strings and I decided it would be easier just to use io.open where I explicitly needed it to handle utf-8 input. This could be a footgun but it works for now.